cognite/client/data_classes/simulators/runs.py

```diff
@@ -232,6 +233,141 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
         )
 
 
+class SimulationValueBase(CogniteObject):
+    """
```

Comment:
I know this is how it's called on the backend, but maybe on client side we should improve the naming:
```
class SimulationValueBase(CogniteObject):
...

class SimulationInput(SimulationValueBase):
...

class SimulationOutput(SimulationValueBase):
       pass
```